### DISCUSSION TITLE
v1.0.0 AI Radio App

### DISCUSSION THREAD

**Discussion by AadrianLeo** [AadrianLeo/AI-Radio-App#1] [General]
@AadrianLeo App is under development.

- You can play different categories of channels.
- you can gave command to the AI to play pause or play particular channel.

<hr /><em>This discussion was created from the release <a href='https://github.com/AadrianLeo/AI-Radio-App/releases/tag/v1.0.0'>v1.0.0 AI Radio App</a>.</em>

**Comment by kUjjwal69** [comment]
bro did you got that late initialization error, due to that radios?? When you were building this project

**Reply by AadrianLeo** [reply]
Can you provide more details about the error with some images will help to understand the error more and solve effectively.

**Comment by kUjjwal69** [comment]
Hello Sir, First of all thanks a lot for replying back, Actually if you see this images, In homepage.dart “the fields radios is unable to fetch data’s from radio.json also if you see the radio.dart file, it’s able to fetch radio.json files. But I don’t know why it is unable to fetch the data from radio.json(In homepage.dart) I am also sending you pubspec.yaml files, In the assests even if I remove assest/radio.json there will be no change, AlsoI updated all my flutter,dart&pubspec Files, But nothing workout  I tried to update my flutter, dart and even my pubspec files but nothing work out. Sent from Mail for Windows From: DHARAMPAL SINGHSent: 14 March 2023 12:10To: AadrianLeo/AI-Radio-AppCc: Ujjwal kumar; CommentSubject: Re: [AadrianLeo/AI-Radio-App] v1.0.0 AI Radio App (Discussion #1) Can you provide more details about the error with some images will help to understand the error more and solve effectively.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Reply by AadrianLeo** [reply]
Thank you for bringing this issue to my attention. I am experiencing a similar problem, as the radio appears to be retrieved but not the associated data. My current hypothesis is that the issue may be related to the URLs contained in the "radio.json" file. It is possible that the author of those URLs has either discontinued them or removed them, as the project and its associated URLs may be outdated. Alternatively, the issue may be caused by some other underlying factor. Furthermore, if you happen to identify a solution to this issue, please do not hesitate to share it with us, as it would greatly assist us in resolving the problem. Rest assured, I am currently investigating the matter and will provide you with a viable solution as soon as possible. In the meantime, I will also explore alternative radio URLs, although this may take some time. Thank you for contacting me regarding this issue. Your feedback is valuable in helping me learn and grow.

**Reply by AadrianLeo** [reply]
I've made several changes to the code to resolve the errors. Most of the issues were caused by outdated dependencies, and as Dart has undergone significant changes, the project was using outdated SDK dependencies.

Regarding the issue with the radio.json file, it appears that the file was able to be fetched, but the data was not being fetched. This problem typically occurs in web apps or Windows apps that work on direct HTTPS. To avoid this, websites should avoid using both HTTP and HTTPS in the same web page, as this can cause mixed content errors that can be a security risk. Browsers have implemented measures to block mixed content by default.

I suggest trying the new code of version 2, and running your Visual Studio Code or any code editor with administrator access. This will ensure that the project runs smoothly and without any issues that due to some linking problem.
